ckanext/lacounts/schemas/dataset.json

```diff
@@ -0,0 +1,156 @@
+        {
+            "field_name": "source_url",
+            "label": "Source URL",
+            "form_snippet": null,
+            "display_snippet": null
+        },
+        {
+            "field_name": "contact_name",
+            "label": "Contact name",
+            "form_placeholder": "e.g. Some Name"
+        },
+        {
+            "field_name": "contact_email",
+            "label": "Email address",
+            "form_placeholder": "e.g. name@domain.com"
+        },
+        {
+            "field_name": "identifier",
+            "label": "Identifier",
+            "form_placeholder": "e.g. <id-example>"
+        },
+        {
+            "field_name": "access_rights",
+            "help_text": "Help text for access rights",
+            "label": "Access Rights",
+            "preset": "select",
+            "choices": [
+                {
+                    "label": "Public",
+                    "value": "public"
+                },
+                {
+                    "label": "Non public",
+                    "value": "non-public"
+                },
+                {
+                    "label": "Restricted",
+                    "value": "restricted"
+                }
+            ]
+        },
+        {
+            "field_name": "license_id",
+            "form_snippet": "license.html",
+            "help_text": "License definitions and additional information can be found at http://opendefinition.org/",
+            "label": "License"
+        },
+        {
+            "field_name": "spatial_text",
+            "help_text": "Help text for text spatial coverage",
+            "label": "Spatial coverage (Text)"
+        },
+        {
+            "field_name": "spatial",
+            "help_text": "Help text for json spatial coverage",
+            "label": "Spatial coverage (GeoJSON)",
+            "preset": "json_object"
+        },
+        {
+            "field_name": "temporal_start",
+            "label": "Temporal coverage start",
+            "preset": "date",
+            "validators": "scheming_required isodate convert_to_json_if_date"
+        },
+        {
+            "field_name": "temporal_end",
+            "label": "Temporal coverage end",
+            "preset": "date",
+            "validators": "scheming_required isodate convert_to_json_if_date"
+        },
+        {
+            "field_name": "temporal",
+            "label": "Temporal coverage",
+            "form_snippet": null,
+            "display_snippet": null
+        },
+        {
+            "field_name": "frequency",
+            "label": "Frequency",
+            "form_placeholder": "e.g. <frequency-example>"
+        },
+        {
+            "field_name": "language",
+            "label": "Language",
+            "form_placeholder": "e.g. <language-example>"
+        },
+        {
+            "field_name": "provenance",
+            "label": "Provenance",
+            "form_placeholder": "e.g. <provenance-example>"
```

Comment:
Make this a textarea, and add the helper_text: "Details about how the data was obtained or processed, who collected it, etc."